src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/model/mapper/EnvelopeMapperTest.java

```diff
@@ -41,22 +45,49 @@ public void should_map_zip_envelope_correctly() throws Exception {
 
         Envelope dbEnvelope = EnvelopeMapper.toDbEnvelope(zipEnvelope, container, Optional.of(ocrValidationWarnings));
 
-        assertThat(dbEnvelope.getCaseNumber()).isEqualTo(zipEnvelope.caseNumber);
-        assertThat(dbEnvelope.getPreviousServiceCaseReference()).isEqualTo(zipEnvelope.previousServiceCaseReference);
-        assertThat(dbEnvelope.getPoBox()).isEqualTo(zipEnvelope.poBox);
-        assertThat(dbEnvelope.getJurisdiction()).isEqualTo(zipEnvelope.jurisdiction);
-        assertThat(dbEnvelope.getDeliveryDate()).isEqualTo(zipEnvelope.deliveryDate);
-        assertThat(dbEnvelope.getOpeningDate()).isEqualTo(zipEnvelope.openingDate);
-        assertThat(dbEnvelope.getZipFileCreateddate()).isEqualTo(zipEnvelope.zipFileCreateddate);
-        assertThat(dbEnvelope.getZipFileName()).isEqualTo(zipEnvelope.zipFileName);
-        assertThat(dbEnvelope.getClassification()).isEqualTo(zipEnvelope.classification);
+        assertSameFields(dbEnvelope, zipEnvelope);
```

Comment:
Please hold back on refactoring so that it's easier for reviewers to identify the actual changes related to the bug/feature in a PR.
Separate PRs are welcome.

Reply:
Done @rkondratowicz